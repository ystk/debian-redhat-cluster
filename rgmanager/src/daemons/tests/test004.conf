<?xml version="1.0"?>
<!-- Change the script, keeping the reference the same.
     the test3 (old) tree should note NEEDSTOP while the new
     tree should note NEEDSTART for the script resource -->
<cluster>
<rm>
	<resources>
    		<service name="test1"/>
		<script name="initscript" file="/etc/init.d/sshd"/>
	</resources>
	<service ref="test1">
		<script ref="initscript"/>
	</service>
</rm>
</cluster>
