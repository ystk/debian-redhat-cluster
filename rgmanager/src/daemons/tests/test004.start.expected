Starting test1...
[start] service:test1
[start] script:initscript
Start of test1 complete
